docs/fsharp/tour.md

```diff
@@ -165,7 +165,7 @@ One special case of Discriminated Union types is the Option Type, which is so us
 
 One unique feature of F#'s type system is the ability to provide context for numeric literals through Units of Measure.
 
-[Units of Measure](language-reference/units-of-measure.md) allow you to associate a numeric type to a unit, such as Meters, and have functions perform work on units rather than numeric literals.  This enables the compiler to verify that the types of numeric literals passed in make sense under a certain context, and eliminate runtime errors associated with that kind of work.
+[Units of Measure](language-reference/units-of-measure.md) allow you to associate a numeric type to a unit, such as Meters, and have functions perform work on units rather than numeric literals.  This enables the compiler to verify that the types of numeric literals passed in make sense under a thus eliminating runtime errors associated with that kind of work.
```

Comment:
This doesn't seem to be grammatically correct. Did you delete the "certain context" part by accident?